### PR TITLE
Add chDB docs

### PR DESCRIPTION
**Pull request by justindeguzman**

### CODE REVIEW

File: docs/en/chdb/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: chDB
+sidebar_label: Overview
+slug: /en/chdb
+description: chDB is an embedded SQL OLAP Engine powered by ClickHouse
+keywords: [chdb, embedded, clickhouse-lite]
+---
+
+# chDB
+
+chDB is an embedded SQL OLAP Engine powered by ClickHouse. It is developed by ClickHouse, Inc and open-source contributors.
```
Comment (auxten) [inline docs/en/chdb/index.md:11]: We need to replace all "embedded" with "in-process"

File: docs/en/chdb/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Library bindings for [Python](https://github.com/chdb-io/chdb), [Go](https://github.com/chdb-io/chdb-go), [Rust](https://github.com/chdb-io/chdb-rust), [NodeJS](https://github.com/chdb-io/chdb-node), [Bun](https://github.com/chdb-io/chdb-bun)
+- Apache License, Version 2.0
+
+## About
+- Read the full story about the birth of the chDB project on Auxten's blog
+- Read about chDB and its usecases on the [Official ClickHouse Blog](https://auxten.com/the-birth-of-chdb/)
+- Discover chDB in your browser using [codapi examples](https://antonz.org/trying-chdb/)
+- Listen to a brief project introduction courtesy of our hero Alexey Milovidov, the original creator of ClickHouse:
+
+<div class='vimeo-container'>
```
Comment (auxten) [inline docs/en/chdb/index.md:28]: There is an offset for the video, you can refer to https://github.com/chdb-io

File: docs/en/chdb/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: chDB
+sidebar_label: Overview
+slug: /en/chdb
+description: chDB is an embedded SQL OLAP Engine powered by ClickHouse
+keywords: [chdb, embedded, clickhouse-lite]
```
Comment (auxten) [inline docs/en/chdb/index.md:6]: add in-process tag

File: docs/en/chdb/install/python.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+python3 -m chdb "SELECT 1, 'abc'" Pretty
+```
+
+Python file example:
+
+```python
+import chdb
+
+res = chdb.query("SELECT 1, 'abc'", "CSV")
+print(res, end="")
```
Comment (auxten) [inline docs/en/chdb/install/python.md:39]: We need introduce more about Session, DataFrame and UDF.
Some doc you can refer:
1. https://github.com/chdb-io/chdb/blob/main/README.md#data-input
1. https://github.com/chdb-io/chdb/blob/f14db2c6b3b8857ba009a20f2c5bb70aa3259a4a/chdb/udf/udf.py#L53
1. https://github.com/chdb-io/chdb/blob/f14db2c6b3b8857ba009a20f2c5bb70aa3259a4a/chdb/session/state.py#L9

File: docs/en/chdb/index.md (hunk 2 above)
Comment (justindeguzman) [reply docs/en/chdb/index.md:28]: I set the offset for the youtube video

**Pull request merged**